nav2_dwb_controller/dwb_core/src/dwb_local_planner.cpp

```diff
@@ -184,6 +192,19 @@ DWBLocalPlanner::loadBackwardsCompatibleParameters()
                                                 //   (local) goal, based on wave propagation
   node_->set_parameters({rclcpp::Parameter("critics", critic_names)});
   /* *INDENT-OFF* */
```

Comment:
I think the indent-off line just goes with the moveParameter lines. I felt they were more readable without wrapping them.

Reply:
Fixed.